charter-ietf-plants.md

```diff
@@ -18,7 +18,7 @@ The PLANTS Working Group will define a mechanism that integrates log constructio
 
 The Working Group will initially put down roots and define the mechanisms needed to interoperably construct and consume certificates:
 
-1. A transparency log structure, maintained by a CA, containing the key/identifier bindings that the CA has certified.
+1. An externally monitorable transparency log structure, maintained by a CA, containing the key/identifier bindings that the CA has certified.
```

Comment:
This prompts a few unanswered questions, but I think that's a good thing (i.e. let's leave them unanswered for the WG):
* External to whom? I think the obvious answer is "the public can monitor", but private PKIs might have a more nuanced answer, so leaving that to the deployment seems solid 
* How is it accessed? Is it the CA that guarantees access or mirrors or both? Does the spec have an opinion on this or do we leave it to deployments?